### ISSUE TITLE
Error on failing run

### ISSUE THREAD

**Issue by archaeogeek** [geopython/GeoHealthCheck#51]
I'm trying to investigate why Geohealthcheck is not sending me any emails and have added an incorrect URL that it should choke on. When I run "python models.py run" manually (virtual environment activated) I can see that it marks the status of the resource as "Changed" but then I get the following error and no email is sent:

Traceback (most recent call last):
  File "models.py", line 250, in <module>
    notify(APP.config, res, run1, last_run_success)
  File "/home/apps/GeoHealthCheck/GeoHealthCheck/notifications.py", line 71, in notify
    msg = render_template2('notification_email.txt', template_vars)
  File "/home/apps/GeoHealthCheck/GeoHealthCheck/util.py", line 121, in render_template2
    translations = translation('messages', 'GeoHealthCheck/translations')
  File "/usr/lib/python2.7/gettext.py", line 484, in translation
    raise IOError(ENOENT, 'No translation file found for domain', domain)
IOError: [Errno 2] No translation file found for domain: 'messages'

So far, I've checked the SMTP settings manually using the smtplib library at a command prompt (lifting the syntax from notifications.py) and as far as I'm concerned that seems to be working fine. The only other change I have made is also to notifications.py, to add a couple of print statements in to find out where in the process the code is choking. No changes have been made to the email template.

**Comment by archaeogeek** [comment]
Just as a further comment, I managed to get this to work by changing line https://github.com/geopython/GeoHealthCheck/blob/master/GeoHealthCheck/util.py#L121 to point at the directory translations rather than GeoHealthCheck/translations. I don't know (yet) whether this means that there's a difference between running the script manually, compared to running it to schedule.

**Comment by justb4** [comment]
Issue confirmed. Will fix. IMO a more reliable fix, independent of GHC working dir is to point to an absolute directory derived from the ``__file__`` var for ``util.py``,  similar to how the ``templates`` dir is determined in line 117:

```
  loader_dir = os.path.join(os.path.dirname(__file__), 'templates')
```

**Comment by archaeogeek** [comment]
I've updated my app to the latest version, to include the fix here, but I'm still not seeing an email. When I run python models.py run I no longer see an error message though. To check I'm doing this right- here's my approach:

1) Create a valid www link to a text file on a remote website (needs to be valid so it can be added to the dashboard)
2) Rename the text file so the www link now triggers a 404 error
3) Go to my GeoHealthCheck app, activate the virtual environment, and from the root directory run:

 `python models.py run`
4) For the 404 link this shows:
```
Testing WWW:LINK http://www.archaeogeek.com/monitor.txt
No handlers could be found for logger "healthcheck"
Adding run
```
No email is sent. I've replicated step 3 from the GeoHealthCheck/GeoHealthCheck directory too, and get the same result. I can confirm emails are working from the server. Am I missing something here?
